src/sentry/tasks/store.py

```diff
@@ -271,7 +257,6 @@ def _do_symbolicate_event(cache_key, start_time, event_id, symbolicate_task, dat
         process_task=process_task,
         data=data,
         data_has_changed=has_changed,
-        new_process_behavior=True,
```

Comment:
Instead of calling `_do_process_event`, can we `submit_process` here now, to fully separate symbolication from processing again?

Reply:
I don't really want to do that until we have a clearer picture of `symbolicate_event/process_event` performance.

Reply:
OK, let's get tests to pass and then I'll approve this.

Reply:
Updated